src/heap.h

```diff
@@ -114,5 +116,92 @@ void Heap<ValueType>::deleteHeap(Node<ValueType>* node) {
     return;
 }
 
+
+template<class ValueType>
+void Heap<ValueType>::show() {}
+
+
+std::string decimalToBinary(int number) {
```

Comment:
A co Ty na to żeby taką metodę wyciągnąć sobie do utilsowej klasy albo do osobnej funkcji? W javie jest sporo takich utilsowych klas. To napewno też trochę porozdziela odpowiedzialności pomiędzy klasami/funkcjami.

Reply:
Heap byłby odpowiedzialny za implementację kopca. A jakaś funkcja/klasa utilsowa za konwersję decimal na binary

Reply:
Takie rozbijanie klas/metod/funkcja ułatwia później testowanie takiego kodu. Jak nam w sumie na koniec zostanie trochę czas, to może moglibyśmy takie testy napisać

Reply:
W sumie już teraz można by było napisac jakieś proste testy który by sprawdzały czy insert + extract działają poprawnie. I ewentualnie za pomocą konsoli wyświetlić info czy test zdany czy nie

Reply:
Można przenieść tą funkcję w sumie.
A co do testów to może jednak jak zostanie czas bo mamy jeszcze dużo innych rzeczy nie zrobione.